src/Oscoin/P2P/Disco/GCE.hs

```diff
@@ -0,0 +1,123 @@
+{-# LANGUAGE DataKinds #-}
+
+module Oscoin.P2P.Disco.GCE where
+
+import           Oscoin.Prelude
+
+import           Oscoin.P2P.Types (Network, renderNetwork)
+
+import           Control.Monad.IO.Unlift (MonadUnliftIO)
+import           Data.HashMap.Strict (HashMap)
+import qualified Data.HashMap.Strict as Map
+import           Data.IP (IP)
+import qualified Data.Set as Set
+import qualified Data.Text as T
+import           Data.Text.Read as T
+import           Lens.Micro
+                 ( Getting
+                 , each
+                 , filtered
+                 , (^?)
+                 , set
+                 , to
+                 , toListOf
+                 , traversed
+                 , _Just
+                 )
+import           Lens.Micro.Extras (view)
+import           Network.Google
+                 (AllowScopes, Env, HasScope, runGoogle, runResourceT, send)
+import           Network.Google.Compute
+                 ( Instance
+                 , InstanceAggregatedList
+                 , InstancesAggregatedList
+                 , NetworkInterface
+                 , i1Labels
+                 , i1NetworkInterfaces
+                 , ialItems
+                 , ialaFilter
+                 , ialiAddtional
```

Comment:
Just for sanity check: is `Addtional` an abbreviation or some editor-smart-completion-propagated typo of `Additional`? 🤔

Reply:
It's fatal to make a typo in a code generator which spits out like 170 libraries 🤷‍♂

Reply:
oh dear, so this comes from the library itself! 😅